seminar/settings.py

```
@@ -140,5 +142,39 @@
+REST_USE_JWT = True  # 🔹 Django에서 JWT 사용을 활성화
+
+SIMPLE_JWT = {
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=30),  # 🔹 Access Token의 유효 기간: 30분
+    'REFRESH_TOKEN_LIFETIME': timedelta(days=1),  # 🔹 Refresh Token의 유효 기간: 1일
+    'ROTATE_REFRESH_TOKENS': True,  # 🔹 Refresh Token을 사용할 때마다 새 토큰 발급
+    'BLACKLIST_AFTER_ROTATION': True,  # 🔹 이전 Refresh Token을 블랙리스트에 추가하여 재사용 방지
+    'AUTH_HEADER_TYPES': ('Bearer',),  # 🔹 인증 헤더 타입을 "Bearer"로 설정
+    'AUTH_TOKEN_CLASSES': ('rest_framework_simplejwt.tokens.AccessToken',),  # 🔹 Access Token 클래스를 지정
+    'ACCESS_TOKEN': 'access_token',  # 🔹 Access Token의 이름 지정
+    'REFRESH_TOKEN': 'refresh_token',  # 🔹 Refresh Token의 이름 지정
+    'BLACKLIST_AFTER_ROTATION': True,
```

Ruff (0.11.9)

Dictionary key literal `'BLACKLIST_AFTER_ROTATION'` repeated

Remove repeated key literal `'BLACKLIST_AFTER_ROTATION'`

(F601)